cmd/podman/generate/kube.go

```diff
@@ -55,7 +55,7 @@ func kube(cmd *cobra.Command, args []string) error {
 	}
 	if cmd.Flags().Changed("filename") {
 		if _, err := os.Stat(kubeFile); err == nil {
-			return errors.Errorf("cannot write to %q", kubeFile)
+			return errors.Errorf("cannot write to %q; file exists", kubeFile)
```

Comment:
This one seems like it doesn't need to be changed, it'snot wrapping anything

Reply:
While looking at the errors I saw this one and wanted to add information.  This error is misleading in that the app could write to the kubeFile, but it won't because the file exists.